src/local/watcher.js

```diff
@@ -233,7 +233,9 @@ class LocalWatcher {
               log.info({path: filePath}, 'file added')
               this.prep.addFileAsync(SIDE, doc).catch(logError)
             } else {
-              const same = find(docs, d => this.pending.hasPath(d.path))
+              const same = find(docs, this.paths
+                  ? d => !fs.existsSync(d.path)
+                  : d => this.pending.hasPath(d.path))
```

Comment:
I believe we should at some point either rename `this.path` to `this.initialPaths` and have `this.isInitialScan = () => this.initialPaths != null`, or even have some kind of `this.initialScan = {paths: []}` object (so we can do `if (this.initialScan) this.initialScan.paths.push(...)` and we don't have a remaining useless `this.initialPaths` property once scan is completed).